Mathlib/RingTheory/Etale/Basic.lean

```diff
@@ -34,8 +34,8 @@ namespace Algebra
 
 section
 
-variable (R : Type u) [CommSemiring R]
-variable (A : Type u) [Semiring A] [Algebra R A]
+variable (R : Type u) [CommRing R]
```

Comment:
Are you intentionally switching from semirings to rings? If so, please write a short reasoning in the PR commit message. I suppose this is because the equivalence with the old definition of formally unramified is only valid in the case of rings, right?

Reply:
Yes. The differential module only makes sense when they are commutative rings.
I have no idea if the original definition makes sense for noncommutative semirings or not anyways.
The PR message is updated.